web3.js/src/fee-calculator.ts

```diff
@@ -1,17 +1,17 @@
-// @flow
+// @ts-ignore
```

Comment:
Assuming no type definitions or BufferLayout?

Reply:
Yeah, unfortunately not. There are some other packages with generated typings that we could switch to later though!